src/test/java/org/elasticsearch/search/MultiValueModeTests.java

```diff
@@ -122,6 +133,9 @@ public int count() {
     private void verify(SortedNumericDocValues values, int maxDoc) {
         for (long missingValue : new long[] { 0, randomLong() }) {
             for (MultiValueMode mode : MultiValueMode.values()) {
+                if (skipModes.contains(mode)) {
```

Comment:
The existing multi-mode tests aren't really capable of testing median properly.  Median currently is cross-tested in the expressions tests, so I would like to hold off on this until the apply/reduce methods are simplified in another PR.